README.rst

```diff
@@ -1 +1,150 @@
-Stub readme.
+edx_event_bus_kafka
+=============================
+
+|pypi-badge| |ci-badge| |codecov-badge| |doc-badge| |pyversions-badge|
+|license-badge|
+
+The ``README.rst`` file should start with a brief description of the repository,
+which sets it in the context of other repositories under the ``edx``
+organization. It should make clear where this fits in to the overall edX
+codebase.
+
+Kafka implementation for Open edX event bus.
+
+Overview (please modify)
+------------------------
+This repository details an implementation of the event bus using Kafka.
+
+Essentially what the event bus does is it processes events from microservices, events can be sent in to the event bus and sent out to be consumed by consumers.
+Events can contain modified data which then send signals to be consumed to their respective consumer via the event bus.
```

Comment:
I really do not understand this sentence, and I'm not entirely sure it's accurate. Signals send events, not the other way around. Also, event data itself should be immutable, although it may contain information about other data modifications that have occurred.
Basically I think we should scrap this sentence.